{
  "input": {
    "sekoiaio": {
      "intake": {
        "dialect": "fortigate",
        "dialect_uuid": "5702ae4e-7d8a-455f-a47b-ef64dd87c981"
      }
    },
    "message": "CEF:0|Fortinet|Fortigate|v6.0.4|32021|event:system login failed|7|deviceExternalId=FGVM2V0000171868 FortinetFortiGatelogid=0100032021 cat=event:system FortinetFortiGatesubtype=system FortinetFortiGatelevel=alert FortinetFortiGatevd=root FortinetFortiGateeventtime=1579172447 FortinetFortiGatelogdesc=Admin login disabled sproc=1.1.1.1 FortinetFortiGateaction=login outcome=failed reason=exceed_limit msg=Login disabled from IP 1.1.1.1 for 60 seconds because of 3 bad attempts"
  },
  "expected": {
    "message": "CEF:0|Fortinet|Fortigate|v6.0.4|32021|event:system login failed|7|deviceExternalId=FGVM2V0000171868 FortinetFortiGatelogid=0100032021 cat=event:system FortinetFortiGatesubtype=system FortinetFortiGatelevel=alert FortinetFortiGatevd=root FortinetFortiGateeventtime=1579172447 FortinetFortiGatelogdesc=Admin login disabled sproc=1.1.1.1 FortinetFortiGateaction=login outcome=failed reason=exceed_limit msg=Login disabled from IP 1.1.1.1 for 60 seconds because of 3 bad attempts",
    "event": {
      "action": "login",
      "code": "0100032021",
      "reason": "exceed_limit",
      "dataset": "event:system",
      "category": "event",
      "outcome": "failed"
    },
    "@timestamp": "2020-01-16T11:00:47Z",
    "log": {
      "level": "alert"
    },
    "observer": {
      "type": "Fortigate",
      "vendor": "Fortinet",
      "version": "v6.0.4"
    },
    "action": {
      "name": "login",
      "type": "system",
      "outcome": "failed",
      "outcome_reason": "Login disabled from IP 1.1.1.1 for 60 seconds because of 3 bad attempts",
      "target": "network-traffic"
    }
  }
}